jinja/index.html

```diff
@@ -50,17 +50,10 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
                 </a>
             </div>
             <div class="col-lg-4 mb-5">
-                <a href="https://pennylane.readthedocs.io">
-                    <div class="card card-image" style="background-image: url(img/tower_small.jpg); background-size: cover;">
-                        <div class="text-center d-flex align-items-center mask rgba-white-strong">
-                            <div>
-                                <h3 class="card-title">
-                                    <strong>Code</strong>
-                                </h3>
-                                <p class="mb-1 black-text px-3">Get started with PennyLane using our quickstart guides, learn how to develop a plugin, and browse the full API.</p>
-                                <div class="white-text d-flex justify-content-center">
-                                    <h5>Documentation <i class="fas fa-angle-double-right"></i></h5>
-                                </div>
+                <a href="https://qhack.ai">
+                    <div class="card card-image" style="background-image: url(img/qhack.png); background-size: cover;">
+                        <div class="text-center d-flex align-items-center mask"  style="background-color: rgba(102, 86, 95, 0.6)">
+                            <div style="width: 100%; min-height: 198px;">
```

Comment:
I don't know, it looks like it doesn't mesh with the other cards. Let's try a few more ideas :thinking:

Reply:
I think the main reason it doesn't mesh is because of how dark the color palette is. I'll try a 'whiter' overlay and see if that makes a difference.

Reply:
![image](https://user-images.githubusercontent.com/2959003/102594860-bcc33c00-4151-11eb-8e90-1eb734a1279e.png)

Reply:
Moving it into the middle: 
![image](https://user-images.githubusercontent.com/2959003/102595001-f09e6180-4151-11eb-824f-98f848d20f92.png)

Reply:
Getting better, but still feels "inserted" somehow and not natural sitting next to the other cards. What if we add some text?

Reply:
I'll give it another go! I think the colour palette of the logo (the hot pink) clashes with the palette of the PL landing page, so its very noticeable

Reply:
Have updated the PR